src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -105,7 +105,10 @@ async function initializeConnection(options: BlazorOptions, logger: Logger): Pro
   });
 
   connection.onclose(error => !renderingFailed && options.reconnectionHandler!.onConnectionDown(options.reconnectionOptions, error));
-  connection.on('JS.Error', error => unhandledError(connection, error, logger));
+  connection.on('JS.Error', error => {
+    renderingFailed = true;
```

Comment:
Do we terminate the circuit in this case?

Reply:
Yes